conda_smithy/configure_feedstock.py

```diff
@@ -2188,17 +2188,17 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
     return config
 
 
-def get_most_recent_version(name):
+def get_most_recent_version(name, only_not_broken=False):
```

Comment:
Shouldn't this be the unconditional default, i.e. not add another parameter?

Reply:
At a minimum, think the default value should be `True`. Choosing to include broken packages should be opt-in (as it is with installing them)

Reply:
Ok, changed the kwarg to `include_broken=False`.